desktop/flipper-ui-core/src/dispatcher/pluginMarketplace.tsx

```diff
@@ -177,7 +184,8 @@ export function autoUpdatePlugins(
       }
     }
   }
-  if (isAutoUpdateDisabled() || !isLoggedIn().get()) {
+  // FIXME: this isLoggedIn blocks the logic
```

Comment:
@nikoant forgot to remove this comment, as we discussed here: https://github.com/facebook/flipper/pull/3473#discussion_r814909991

So either I can push changes and remove the comment and change the stub value, or if already pulled to internal codebase, you could do the change there. Let me know which works better for you.